src/main/java/duke/UiManager.java

```diff
@@ -0,0 +1,120 @@
+package duke;
+import duke.task.Task;
```

Comment:
Try to be consistent. You added an empty line between package and import in the TaskList.java file and other files, but not here :)